examples/ddpg/ddpg.py

```diff
@@ -10,63 +10,45 @@
 from torchrl.envs.utils import set_exploration_mode
 from torchrl.record import VideoRecorder
 
-try:
-    import configargparse as argparse
-
-    _configargparse = True
-except ImportError:
-    import argparse
-
-    _configargparse = False
+import hydra
```

Comment:
@jieru-hu just double checking, we wan t to use `hydra-core >=1.2` when developing on our end but for specifying the dependency, we should mention `hydra-core>=1.0` correct? This should make it easier for people to use Hydra + Torch RL

Reply:
@shagunsodhani  - that's a good point. However if we only specify hydra-core >= 1.0 then we cannot really use more advanced feature in  1.1 and 1.2... (recursive defaults lists in 1.1 etc, but maybe those are not needed here?)
i think it should safe to depends on `>= 1.1` since Hydra 1.1 has been out for almost a year :)

Reply:
Sounds good! Lets use `hydra-core>=1.1` in dependencies.

Reply:
https://github.com/BoboBananas/rl/pull/3/commits/f986e2dd3e1181e03ae94c28d9736c14584dccca